gordo_client/io.py

```diff
@@ -36,6 +36,14 @@ class BadGordoRequest(Exception):
     pass
 
 
+class BadGordoResponse(Exception):
+    """
+    Represents a general bad response (not json or model)
+    """
+
+    pass
```

Comment:
`pass` is redundant: 

```suggestion
class BadGordoResponse(Exception):
    """
    Represents a general bad response (not json or model)
    """
```

Reply:
fixed